<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpDatabaseProtocolDriverUI</class>
 <widget class="QWidget" name="UdpDatabaseProtocolDriverUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>307</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="9" column="4">
     <widget class="QPushButton" name="pbClose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QLabel" name="lbState">
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="lbReceiveDataN">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="lbAcceptData">
      <property name="text">
       <string>有效数据：
(测点数)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbReceiveData">
      <property name="text">
       <string>接受数据：
(UDP包数)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="lbAcceptDataN">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pbStart">
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pbStop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="3">
     <widget class="QLabel" name="lbLastError">
      <property name="text">
       <string>最后错误：无</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QComboBox" name="cbInterface"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbHostPort">
      <property name="text">
       <string>HostPort</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbPort">
      <property name="text">
       <string>LocalPort</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbInterface">
      <property name="text">
       <string>选择组播网卡</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="leLocalPort"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="lbStationID">
      <property name="text">
       <string>StationID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="leStationID"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="leHostPort">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="lbHostIP">
      <property name="text">
       <string>HostIP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="leHostIP">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
